proto/sync/sync.proto

```diff
@@ -32,62 +60,103 @@ message ChangeProofRequest {
   uint32 bytes_limit = 6;
 }
 
-// KeyValue represents a single key and its value.
-message KeyValue {
-  bytes key = 1;
-  bytes value = 2;
+message GetChangeProofRequest {
+  bytes start_root_hash = 1;
+  bytes end_root_hash = 2;
+  bytes start_key = 3;
+  bytes end_key = 4;
+  uint32 key_limit = 5;
 }
 
-// KeyChange is a change for a key from one revision to another.
-// If the value is None, the key was deleted.
-message KeyChange {
-  bytes key = 1;
-  MaybeBytes value = 2;
+// TODO improve naming. This is the response to a sync request
+// for a change proof, but it's not a response to a GetChangeProof
+// request on the SyncableDB service. That's confusing.
+message GetChangeProofResponse {
```

Comment:
Why isn't this the type of the GetChangeProof result on the service?

Reply:
The `SyncableDB` service implements `sync.SyncableDB`, and the `GetChangeProof` method on that interface always returns just a change proof (not a range proof).  Note that `SyncableDB` isn't the sync client/server interface.